src/OSPSuite.Core/Domain/Builder/IndividualBuildingBlock.cs

```diff
@@ -0,0 +1,11 @@
+﻿using OSPSuite.Utility.Collections;
+
+namespace OSPSuite.Core.Domain.Builder
+{
+   public class IndividualBuildingBlock : PathAndValueEntityBuildingBlockFromPKSim<IndividualParameter>
+   {
+      public Cache<string, string> OriginData { set; get; } = new Cache<string, string>();
```

Comment:
@rwmcintosh As mentiioned in the chat, I think this is not the right abstraction here